scripts/prebuild.sh

```diff
@@ -1,11 +1,20 @@
 #!/bin/bash
-if [ ! -z "$VERCEL_GIT_COMMIT_REF" ]
+if [ ! -z "$VERCEL_GIT_COMMIT_REF" ] 
 then
   npm install --quiet -g @takeshape/cli
-  takeshape branch delete --name ${VERCEL_GIT_COMMIT_REF}
-  takeshape branch create --name ${VERCEL_GIT_COMMIT_REF}
-  takeshape import --schema --preserveAuth --branch ${VERCEL_GIT_COMMIT_REF} --from .takeshape/pattern/schema.json
-  echo `takeshape branch versionUrl --name ${VERCEL_GIT_COMMIT_REF}` > .current-branch-url
+
+  # Main branch
+  if [ "$VERCEL_GIT_COMMIT_REF" == "main" ]; then
+    echo `takeshape branch tagVersion --tag ${VERCEL_GIT_COMMIT_SHA}` > .current-branch-url
+
+  # There is a takeshape branch that matches this git branch
+  elif takeshape branch list | grep -q "📁 $VERCEL_GIT_COMMIT_REF (development)"; then
```

Comment:
```suggestion
  elif takeshape branch url --name ${VERCEL_GIT_COMMIT_REF}; then
```
I think this approach is cleaner. Need this PR to land first though: https://github.com/takeshape/takeshape/pull/3813